src/main/scala/tech/cryptonomic/conseil/metadata/AttributeValuesCacheConfiguration.scala

```diff
@@ -16,6 +16,14 @@ class AttributeValuesCacheConfiguration(metadataConfiguration: MetadataConfigura
         .flatMap(_.cacheConfig)
     }.flatten
 
+  /** extracts cache configuration for given attribute path */
+  def getCardinalityHint(path: AttributePath): Option[Int] =
+    when(metadataConfiguration.isVisible(path)) {
```

Comment:
What about introducing a `whenVisible(path)(optionResult)` local method to lighten up these calls?

Reply:
good idea, added